preload/delaying_onload_link_preload_after_discovery.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+    var t = async_test('Makes sure link preload preloaded resources are delaying onload after discovery');
+</script>
+<link rel=preload href="resources/dummy.js?pipe=trickle(d1)" as=script>
+<link rel=preload href="resources/square.png?pipe=trickle(d1)" as=image>
+<script>
+    window.addEventListener("load", t.step_func(function() {
+        assert_equals(performance.getEntriesByType("resource").length, 4);
+        t.done();
+    }));
+    document.write('<sc' + 'ript src="resources/dummy.js?pipe=trickle(d1)"></sc' + 'ript>');
+    document.write("<!--");
```

Comment:
What's the point/purpose of the unclosed comment? Would it be simpler and cleaner to simply build and append a script element?